prov/gni/src/gnix_fabric.c

```diff
@@ -338,26 +236,98 @@ static int gnix_getinfo(uint32_t version, const char *node, const char *service,
 	gnix_info->rx_attr->size = GNIX_RX_SIZE_DEFAULT;
 	gnix_info->rx_attr->iov_limit = GNIX_MAX_MSG_IOV_LIMIT;
 
+	return gnix_info;
+}
+
+static void _gnix_freeinfo(struct fi_info *info)
```

Comment:
Does this mean that when a user calls fi_freeinfo, we may leak the src_addr and dest_addr fields?